DiamondLightSource/SynchWeb
# LIMS-1206: Remove stacked usage bar chart from visit page

**JIRA ticket**: [LIMS-1206](https://jira.diamond.ac.uk/browse/LIMS-1206)

**Summary**:

At the top of each visit page, there is a coloured bar showing how time was used. The same info is shown on the Visit Stats page as a pie chart. Showing it on the visit page just makes the page look busier and is an extra call to the API. 

**Changes**:
- Remove the stacked bar chart

**To test**:
- Go to a visit page, check the stacked bar chart is not there
- Check the rest of the data collections are shown as usual
